src/Makefile

```diff
@@ -245,7 +245,8 @@ ifeq ($(PLATFORM),PLATFORM_DRM)
 endif
 ifeq ($(PLATFORM),PLATFORM_WEB)
     # On HTML5 OpenGL ES 2.0 is used, emscripten translates it to WebGL 1.0
-    GRAPHICS = GRAPHICS_API_OPENGL_ES2
+    #GRAPHICS = GRAPHICS_API_OPENGL_ES2
```

Comment:
I prefer to keep `GRAPHICS_API_OPENGL_ES2` as the default, instead of `GRAPHICS_API_OPENGL_ES3`

Reply:
Regarding your 3 comments: my bad! They were not intended to be part of the PR. Those are changes I made afterwards that I require for my project but are not meant to be merged into raylib. Switching by default to ES3 would break existing projects indeed.